src/Checkbox/__tests__/__snapshots__/Checkbox.test.js.snap

```diff
@@ -70,19 +70,14 @@ exports[`<Checkbox /> properly renders a checkbox 1`] = `
 >
   <input
     aria-invalid={false}
-    checked={false}
```

Comment:
are the changes in this file just explicit defaults being removed?

Reply:
very similar to what I ran into with the `css` prop in the monolith

Reply:
basically it looks like the base HTML element props are being stripped

Reply:
Similar to the above, I just removed `false` defaults that could just as well be left `undefined`.  I'm happy to put the defaults back if there's a concern.

Reply:
I'm going to put `checked` back since it feels wrong leaving it undefined.

Reply:
Actually I'll just restore the defaults so the snapshots are unchanged.  Better to minimize the chances of unexpected behavior changes to start with.  We can always update the prop defaults later.